Backpack-SwiftUI/Chip/Classes/BPKDismissableChip.swift

```diff
@@ -0,0 +1,126 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+public struct BPKDismissableChip: View {
+    class DismissableChipViewModel: ObservableObject {
```

Comment:
This is kind of a "hack", as otherwise updating state from within a view directly is not reliable (sometimes it works, sometimes it doesn't)
Xcode even shows a runtime warning (the violet one) saying that this is not recommended, but there's just no other way to achieve this dismissable chip behaviour.